upgrading/topics/keycloak/changes.adoc

```diff
@@ -1,5 +1,13 @@
 == Migration Changes
 
+=== Migration to 9.0.1
```

Comment:
Is it possible to remove this? We're still keeping the WebAuthn in the project enabled by default as mentioned above.